src/Files.App/Filesystem/WSLDistroManager.cs

```diff
@@ -59,6 +60,14 @@ public async Task UpdateDrivesAsync()
 			}
 		}
 
+		public bool TryGetDistro(string path, out WslDistroItem? distro)
+		{
+			var normalizedPath = PathNormalization.NormalizePath(path);
+			distro = Distros.FirstOrDefault(x => path.StartsWith(PathNormalization.NormalizePath(x.Path), StringComparison.OrdinalIgnoreCase));
```

Comment:
Here there's something wrong: you never read the value assigned to `normalizedPath`

```suggestion
			var normalizedPath = PathNormalization.NormalizePath(path);
			distro = Distros.FirstOrDefault(x => normalizedPath.StartsWith(PathNormalization.NormalizePath(x.Path), StringComparison.OrdinalIgnoreCase));
```